src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/monetizationmodule/MonetizationModuleComponentObject.java

```diff
@@ -190,14 +190,10 @@ public void verifyAdsenseUnitNotShownAboveFooter() {
              true);
   }
 
-  public boolean verifyWindowWidth(int width) {
-    if (getWindowSize().width != width) {
-      PageObjectLogging.log("verifyWindowWidth",
-                            "The width of the window is invalid (width="
-                            + getWindowSize().width + ")", true);
-      return false;
-    }
-    return true;
+  public boolean verifyWindowWidth(int expectedWidth) {
+    int windowWidth = getWindowSize().width;
+    PageObjectLogging.log("verifyWindowWidth", "The width of the window is " + windowWidth, true);
```

Comment:
The previous code only logged on error - is there any reason to log if there's no problem?  I'm not a selenium expert, but I've always preferred tests that are emit nothing unless there's a problem because the informational messages make it harder to find the problems.  Do selenium tests typically log informational messages?